life_expectancy/cleaning.py

```diff
@@ -0,0 +1,64 @@
+"""Module to clean dataframe"""
+import sys
+from typing import List
+
+import pandas as pd
+
+from .loaders import load_data, save_data
+
+def _split_comma_separated_column(data: pd.DataFrame, column:int) -> pd.DataFrame:
+    data[data.columns[column].split(",")] = [x.split(',') for x in data.iloc[:, column]]
+    return data
+
+def _unpivot_year_columns(data: pd.DataFrame) -> pd.DataFrame:
+    years_cols = [x for x in data.columns if x.strip().isnumeric()]
+    data = pd.melt(data, id_vars=data.columns[0], value_vars=years_cols, \
+        value_name='value', var_name='year')
+    return data
+
+def _drop_backslash_from_column_name(data:pd.DataFrame, column:str) -> pd.DataFrame:
+    return data.rename(columns={column: column.partition('\\')[0]})
+
+def _filter_dataframe(data:pd.DataFrame, country:str) -> pd.DataFrame:
+    return data.loc[data.region.str.upper() == country.upper()]
+
+def clean_data(
+        argv : List[str],
```

Comment:
this name is not explicit at all and there is no documentation in the docstring for the reader to understand what it contains. I literally have to scan the function definition to know what it should contain